pkg/inventory/inventory-client_test.go

```diff
@@ -571,7 +571,7 @@ func TestMergeInventoryObjs(t *testing.T) {
 				}
 				wrapped := WrapInventoryObj(retained)
 				mergedObjs, _ := wrapped.Load()
-				if !object.SetEquals(tc.expected, mergedObjs) {
+				if !object.IsSetEqual(tc.expected, mergedObjs) {
```

Comment:
Remove this change. If you want to advocate for a name change, please do it in another PR.